### ISSUE TITLE
Improve accessibility of the navigation

### ISSUE THREAD

**Issue by BlueCutOfficial** [DazzlingFugu/ember-fr-guides-source#222]
We miss a couple of things so screen readers give feedback when the navigation is finished. We should have a look at the latest Ember Guides as it's a different website that is updated independently and they improved this with their new layout.
